packages/core/helper-plugin/lib/src/components/DateTimePicker/tests/index.test.js

```diff
@@ -5,6 +5,9 @@ import { lightTheme } from '@strapi/design-system/themes';
 import DateTimePicker from '../index';
 
 describe('DateTimePicker', () => {
+  jest.useFakeTimers('modern');
+  jest.setSystemTime(new Date(2022, 2, 2));
```

Comment:
@petersg83 This isn't related to this PR, but it hopefully fixes the flaky snapshot test we saw during the last couple of days :)